addons/mail/views/res_users_views.xml

```diff
@@ -35,12 +35,14 @@
                     <field name="signature" position="attributes">
                         <attribute name="widget">html_mail</attribute>
                     </field>
-                    <field name="signature" position="after">
-                        <field name="out_of_office_from" widget="daterange"
-                               options="{'end_date_field': 'out_of_office_to'}"/>
-                        <field name="out_of_office_to" invisible="1"/>  <!-- otherwise it is visible, smort -->
-                        <field name="out_of_office_message"/>
-                    </field>
+                    <group name="calendar_preferences" position="inside">
+                        <label for="out_of_office_to" string="Out-of-office" />
+                        <div id="full_date" class="o_row">
+                            <field name="out_of_office_to" widget="daterange" options="{'end_date_field': 'out_of_office_from', 'show_time': false}" placeholder="None planned"/>
+                            <field name="out_of_office_from" invisible="1"/>
```

Comment:
I am willing to bet that if there was a comment before then you will need to add it back otherwise the Runbot won't be happy about it. => That's the new policy since the new handling of invisible fields in views.